### PR TITLE
Warn info screen (EXPOSUREAPP-8947)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
Warn guests info screen.

Screenshot 2021-08-12 at 17 57 31 Screenshot 2021-08-12 at 17 57 07

How to test:
1) Home Screen -> "Planning an event?" card
2) Three point menu -> "In Vertretung warnen" 
3) ✨ Tadaaa ✨

### CODE REVIEW

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by mtwalli — COMMENTED** [review]
Very good 👍 , I don't think we need settings in this case

File: Corona-Warn-App/src/main/res/layout/trace_location_organizer_warn_info_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/res/layout/trace_location_organizer_warn_info_fragment.xml:1]: could you call "Reformat Code" here?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/warn/info/TraceLocationWarnInfoViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.ui.presencetracing.organizer.warn.info
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/warn/info/TraceLocationWarnInfoViewModel.kt:1]: Is this necessary? We have a static fragment with one button only for navigation to the next fragment

**Review by jurajkusnier — COMMENTED** [review]
As @mtwalli mentioned we don't need settings.
IMHO in that case code could be much simpler just one static fragment no AutoInject, no ViewModel, no new modules

**Review by jurajkusnier — APPROVED** [review]
Nice 👍

**Review by mtwalli — APPROVED** [review]

**Pull request merged**